docs/src/reference/gremlin-applications.asciidoc

```diff
@@ -942,8 +942,10 @@ The following table describes the various YAML configuration options that Gremli
 |authentication.authenticator |The fully qualified classname of an `Authenticator` implementation to use.  If this setting is not present, then authentication is effectively disabled. |`AllowAllAuthenticator`
 |authentication.authenticationHandler | The fully qualified classname of an `AbstractAuthenticationHandler` implementation to use. If this setting is not present, but the `authentication.authenticator` is, it will use that authenticator with the default `AbstractAuthenticationHandler` implementation for the specified `Channelizer` |_none_
 |authentication.config |A `Map` of configuration settings to be passes to the `Authenticator` when it is constructed.  The settings available are dependent on the implementation. |_none_
-|authentication.enableAuditLog |The available authenticators can issue audit logging messages, binding the authenticated user to his remote socket address and binding requests with a gremlin query to the remote socket address. For privacy reasons, the default value of this setting is false. The audit logging messages are logged at the INFO level via the `audit.org.apache.tinkerpop.gremlin.server` logger, which can be configured using the log4j.properties file. |false
+|authentication.authorizer |The fully qualified classname of an `Authorizer` implementation to use.  Gremlin server does not offer `Authorizer` implementations, but link:https://tinkerpop.apache.org/docs/x.y.z/dev/provider/#_authentication_and_authorization[graph system providers can choose to do so]. |_none_
```

Comment:
> graph system providers can choose to do so

is it just up to graph system providers? couldn't users write them as well?

Reply:
Indeed, if we document in the ref docs we do not need the provider term any more.